tests/unit/rpc/handlers/AccountInfoTests.cpp

```diff
@@ -655,6 +647,48 @@ TEST_F(RPCAccountInfoHandlerTest, IdentAndSignerListsFalse)
     });
 }
 
+TEST_F(RPCAccountInfoHandlerTest, EmptySignerLists)
+{
+    auto const ledgerHeader = createLedgerHeader(kLEDGER_HASH, 30);
+    EXPECT_CALL(*backend_, fetchLedgerBySequence).WillOnce(Return(ledgerHeader));
+
+    auto const account = getAccountIdWithString(kACCOUNT);
+    auto const accountKk = ripple::keylet::account(account).key;
+    auto const accountRoot = createAccountRootObject(kACCOUNT, 0, 2, 200, 2, kINDEX1, 2);
+    ON_CALL(*backend_, doFetchLedgerObject(accountKk, 30, _))
+        .WillByDefault(Return(accountRoot.getSerializer().peekData()));
```

Comment:
Is it expected to be called more than once? I think it would be better to put a specific number of expected calls here because we should fetch data from db as few times as possible.

Reply:
I think it would still better to check that we don't query DB more times than it's actually needed.